### PR TITLE
Create Disable or enable journaling of voice mail and missed call not…

### PR DESCRIPTION
**Pull request by Tangela123**
…ifications

### CODE REVIEW

**Comment by cabailey** [comment]
@JohanFreelancer9 This is to recreate https://github.com/MicrosoftDocs/OfficeDocs-Exchange/pull/2197 that got mistakenly closed & couldn't be reopened.  Please help to see this through and merge so it can be published.

File: Exchange/ExchangeOnline/security-and-compliance/journaling/Disable or enable journaling of voice mail and missed call notifications [hunk 1]
```diff
@@ -0,0 +1,30 @@
+In Exchange Online, when you create a journal rule to journal email messages sent to or from recipients or senders in an Exchange organization, voice mail and missed call notifications generated by the Unified Messaging (UM) service are included. Use the procedures in this topic to turn this feature on or off for your entire organization.
+
+ [!IMPORTANT]
+> When voice mail journaling is disabled all Unified Messages, which contain the following messages classes will not be journaled. It is important to be aware that messages that are “spoof” using these message class will not be journaled.”
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/journaling/Disable or enable journaling of voice mail and missed call notifications:4]:
```suggestion
> When voice mail journaling is disabled, all Unified Messages that contain the following message classes will not be journaled. It is important to be aware that messages that are “spoof” will not be journaled when using these message classes.
```

File: Exchange/ExchangeOnline/security-and-compliance/journaling/Disable or enable journaling of voice mail and missed call notifications [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+"IPM.Note.Microsoft.Voicemail.UM"
+"IPM.Note.Microsoft.Voicemail.UM.CA"
+"IPM.Note.Microsoft.Missed.Voice"
+"IPM.Note.rpmsg.Microsoft.Voicemail.UM.CA"
+"IPM.Note.rpmsg.Microsoft.Voicemail.UM" 
+What do you need to know before you begin?
+
+Estimated time to complete: 5 minutes.
+
+You can only use the PowerShell to perform this procedure.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/journaling/Disable or enable journaling of voice mail and missed call notifications:15]:
```suggestion
You can only use PowerShell to perform this procedure.
```

File: Exchange/ExchangeOnline/security-and-compliance/journaling/Disable or enable journaling of voice mail and missed call notifications [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Use the PowerShell to disable or enable journaling of voice mail and missed call notifications
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/security-and-compliance/journaling/Disable or enable journaling of voice mail and missed call notifications:17]:
```suggestion
Use PowerShell to disable or enable journaling of voice mail and missed call notifications.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review is done - I cannot see the rich diff for some reason, so could not check formatting. Unfortunately, I also do not have merge rights.

**Comment by Tangela123** [comment]
The Article is still not showing up in the Table of contents

**Pull request merged**